apps/docs-app/sidebars.js

```diff
@@ -195,6 +195,11 @@ const sidebars = {
           id: 'guides/compatibility',
           label: 'Version Compatibilty',
         },
+        {
+          type: 'doc',
```

Comment:
This should go into a different section for developers. This isn't necessarily for end users